isis/src/control/objs/BundleAdjust/BundleAdjust.cpp

```diff
@@ -1896,234 +1880,69 @@ namespace Isis {
-                                               &coeffTarget(1, index));
-      index++;
-    }
-
-    if (m_bundleSettings->solveTargetBody() && m_bundleTargetBody->solveTriaxialRadii()) {
-
-      std::vector<double> lookBWRTRadiusA =
-          measureCamera->GroundMap()->EllipsoidPartial(surfacePoint,
-                                                       CameraGroundMap::WRT_MajorAxis);
-
-      measureCamera->GroundMap()->GetdXYdPoint(lookBWRTRadiusA, &coeffTarget(0, index),
-                                               &coeffTarget(1, index));
-      index++;
-
-      std::vector<double> lookBWRTRadiusB =
-          measureCamera->GroundMap()->EllipsoidPartial(surfacePoint,
-                                                       CameraGroundMap::WRT_MinorAxis);
-
-      measureCamera->GroundMap()->GetdXYdPoint(lookBWRTRadiusB, &coeffTarget(0, index),
-                                               &coeffTarget(1, index));
-      index++;
-
-      std::vector<double> lookBWRTRadiusC =
-          measureCamera->GroundMap()->EllipsoidPartial(surfacePoint,
-                                                       CameraGroundMap::WRT_PolarAxis);
-
-      measureCamera->GroundMap()->GetdXYdPoint(lookBWRTRadiusC, &coeffTarget(0, index),
-                                               &coeffTarget(1, index));
-      index++;
-    }
-
-    index = 0;
-
-    if (observationSolveSettings->instrumentPositionSolveOption() !=
-        BundleObservationSolveSettings::NoPositionFactors) {
-
-      int numCamPositionCoefficients =
-          observationSolveSettings->numberCameraPositionCoefficientsSolved();
-
-      // Add the partial for the x coordinate of the position (differentiating
-      // point(x,y,z) - spacecraftPosition(x,y,z) in J2000
-      for (int cameraCoef = 0; cameraCoef < numCamPositionCoefficients; cameraCoef++) {
-        measureCamera->GroundMap()->GetdXYdPosition(SpicePosition::WRT_X, cameraCoef,
-                                                    &coeffImage(0, index),
-                                                    &coeffImage(1, index));
-        index++;
-      }
-
-      // Add the partial for the y coordinate of the position
-      for (int cameraCoef = 0; cameraCoef < numCamPositionCoefficients; cameraCoef++) {
-        measureCamera->GroundMap()->GetdXYdPosition(SpicePosition::WRT_Y, cameraCoef,
-                                                    &coeffImage(0, index),
-                                                    &coeffImage(1, index));
-        index++;
+    if (measureCamera->GetCameraType() != 6) {
+      // Compute the look vector in instrument coordinates based on time of observation and apriori
+      // lat/lon/radius.  As of 05/15/2019, this call no longer does the back-of-planet test. An optional
+      // bool argument was added CameraGroundMap::GetXY to turn off the test.
+      double computedX, computedY;
+      if (!(measureCamera->GroundMap()->GetXY(point.adjustedSurfacePoint(),
```

Comment:
These are never used here. This entire if block should go.

Reply:
Could we actually chat about this post-standup. It does change the state of the camera ground map, and breaks everything if I pull it out. I'm concerned about the extra time it might take to run this in each individual "compute partials" function, but haven't tested this yet.

Reply:
Discussed in person, this is actually needed for Isis observations, so it's going to stay in.